hazelcast/src/main/java/com/hazelcast/util/MapUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.hazelcast.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utility class for Maps
+ */
+public final class MapUtil {
+
+    private static final double HASHMAP_DEFAULT_LOAD_FACTOR = 0.75;
+
+    private MapUtil() { }
+
+    /**
+     * Utility method that creates an {@link java.util.HashMap} with its initialCapacity calculated
+     * to minimize rehash operations
+     */
+    public static <K, V> Map<K, V> createHashMap(int expectedMapSize) {
+        int initialCapacity = (int) (expectedMapSize * HASHMAP_DEFAULT_LOAD_FACTOR) + 1;
```

Comment:
Seems it should be `/` instead of `*`
`int initialCapacity = (int) (expectedMapSize / HASHMAP_DEFAULT_LOAD_FACTOR) + 1`